common/src/Model/GameFactory.h

```diff
@@ -102,6 +102,9 @@ namespace TrenchBroom {
             bool setGamePath(const std::string& gameName, const IO::Path& gamePath);
             bool isGamePathPreference(const std::string& gameName, const IO::Path& prefPath) const;
 
+            IO::Path compilationToolPath(const std::string& gameName, const std::string& toolName) const;
```

Comment:
I wonder why these functions are in `GameFactory`. I don't really have a better place to put them (maybe a free function in `GameConfig`?), but this seems an odd place. Or maybe `GameFactory` is a bad name. I mean, it is. But maybe it could be better.

Reply:
I put them here because they're very similar to the game path. So if we move one, we should move the other.

I'm not sure where they would move to... could be left for another PR.

Reply:
Yeah, let's leave that for another PR then.